components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -431,48 +435,7 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
                 }
                 if (!this.state.desktopIde) {
                     phase = StartPhase.Running;
-
```

Comment:
We can delete this condition because it has already been redirected

Reply:
If we are going to use `redirect` for running phase, we can remove this safely. If not, we can change it @geropl